SniffMeet/SniffMeet/Source/Common/BaseView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BaseView.swift
+//  SniffMeet
+//
+//  Created by sole on 11/12/24.
+//
+
+import UIKit
+
+class BaseView: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        backgroundColor = .white
+        
+        configureAttributes()
+        configureHierarchy()
+        configureConstraints()
+        bind()
+    }
+    
+    /// frame이 zero로 설정됩니다.
+    init() {
+        super.init(frame: .zero)
+
+        backgroundColor = .white
+
+        configureAttributes()
+        configureHierarchy()
+        configureConstraints()
+        bind()
+    }
```

Comment:
convenience init 어떤가요?
```suggestion
convenience init() {
    self.init(frame .zero)
}
```

Reply:
고것도 고려를 해보았는데, 
convenience가 수평적으로 확장되는 생성자(like extension)라서 상속시 저 생성자를 사용할 수 없더라구요... 그래서 지정생성자로 만들어줬습니다~~

Reply:
오버라이드 된 이니셜라이저한태는 사용이 안되나 보군요...ㅋㅋ